Content.Server/Dragon/DragonComponent.cs

```diff
@@ -0,0 +1,72 @@
+using Content.Shared.Actions;
+using Content.Shared.Actions.ActionTypes;
+using Robust.Shared.Containers;
+using System.Threading;
+
+namespace Content.Server.Dragon
+{
+    [RegisterComponent]
+    public sealed class DragonComponent : Component
+    {
```

Comment:
for the future we use filescoped namespaces now so you dont need to have a namespace block